src/components/CommentTeaser/CommentTeaser.js

```diff
@@ -127,12 +120,23 @@ export const CommentTeaser = ({
     Link
   }
 
+  const linkEMRule = useMemo(
```

Comment:
```suggestion
  const highlightEMRule = useMemo(
```

Took me a while to understand that it's only needed for `CommentBodyParagraph` which supports highlight html  from elastic search with em tags.